src/EnergyPlus/SimAirServingZones.hh

```diff
@@ -153,7 +153,11 @@ namespace SimAirServingZones {
                              bool FirstHVACIteration,     // TRUE if first full HVAC iteration in an HVAC timestep
                              int AirLoopNum,              // Primary air loop number
                              int &CompIndex,              // numeric pointer for CompType/CompName -- passed back from other routines
-                             HVACSystemData *CompPointer);
+                             HVACSystemData *CompPointer, // equipment actual pointer
+                             int const &airLoopNum,       // index to AirloopHVAC
+                             int const &branchNum,        // index to AirloopHVAC branch
+                             int const &compNum           // index to AirloopHVAC branch component
+    );
```

Comment:
Since this passes a copy of CompPointer, added last 3 arguments so CompPointer could be directly set inside this function in PrimaryAirSystem struct.